mono/metadata/object.c

```diff
@@ -5391,8 +5390,10 @@ mono_object_new_common_tail (MonoObject* o, MonoClass* klass, MonoError* error)
 static MonoObjectHandle
```

Comment:
In most c coding conventions including ours,. the '*' is next to the variable name, not the type.

Reply:
Fixed a few.